### ISSUE TITLE
Notifications not working anymore

### ISSUE THREAD

**Issue by francoism90** [laravel-filament/filament#8716]
### Package

filament/filament

### Package Version

v3.0.61

### Laravel Version

v10.24.0

### Livewire Version

v3.0.5

### PHP Version

PHP 8.2

### Problem description

After upgrading to v3.0.61, notifications stopped working.

### Expected behavior

Notifications working

### Steps to reproduce

Upgrade to latest version

### Reproduction repository

https://github.com/francoism90/hub-api

### Relevant log output

```shell
Uncaught TypeError: _t is undefined
    H https://example.test/js/filament/notifications/notifications.js?v=3.0.61.0:5

[..]

Alpine Expression Error: notificationComponent is not defined
```

**Comment by underdpt** [comment]
I can confirm this is happening after upgrading to 3.0.61. Notifications work on 3.0.60.

**Comment by danharrin** [comment]
Released v3.0.62
